tests/isolated/patcher_fork_after_monkey_patch.py

```diff
@@ -31,23 +31,23 @@ def target():
     for t in threads:
         t.start()
 
-    check(2, threading, 'pre-fork patched')
+    check(5, threading, 'pre-fork patched')
     check(3, _threading, 'pre-fork original')
-    check(4, eventlet.green.threading, 'pre-fork green')
+    check(5, eventlet.green.threading, 'pre-fork green')
 
     if os.fork() == 0:
```

Comment:
This was asserting that the child lost all its threads. Which would be the case for real threads, but green threads _do_ survive fork() so these assertions were wrong.

Reply:
Ack, thanks for the heads up. Make sense.